src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +21,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(s -> Arrays.stream(s.split(NUMBERS_SEPARATOR)))
+                .mapToInt(Integer::parseInt)
+                .filter(i -> i % 2 == 0)
```

Comment:
For the future, do not use short names that are difficult to read
```suggestion
                .flatMap(stringNumbers -> Arrays.stream(stringNumbers.split(NUMBERS_SEPARATOR)))
                .mapToInt(Integer::parseInt)
                .filter(number -> number % 2 == 0)
```